README.md

```diff
@@ -1,2 +1,47 @@
 # Greetings
-A site that displays: "Hello World"
+A website that displays the text"Hello World"
+
+https://user-images.githubusercontent.com/62938735/124670432-e5234580-deab-11eb-975d-c795c2384459.jpg
+
+It displays a blue hello world text, centred vertically and horizontally using margin and text-align properties in CSS 
+
+**Built With**
+
+●HTML
+
+●CSS
+
+●Linters
+
+**Live Demo**
+
+https://www.loom.com/share/16dbc9ea55754157985d307136558780
+
+**Getting Started**
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
Good job 👍🏽 

- [ ] kindly give more details on how to get a local copy on your machine and run

You can example like this:

- [ ] clone the app by running this command ```git clone your-github-url```
- [ ] ```cd your-github-url```